docs/overview.md

```diff
@@ -0,0 +1,184 @@
+# Overview
+
+This SDK has several entry points:
+
+- You can run the application using the [CLI](./src/CarbonAware.CLI) and refer
+  to more documentation [here](./carbon-aware-cli.md).
+
+- You can build a container containing the [WebAPI](./src/CarbonAware.WebApi)
+  and connect via REST requests and refer to more documentation
+  [here](./carbon-aware-webapi.md).
+
+- (Future) You can install the Nuget package and make requests directly.
+  ([tracked here](https://github.com/Green-Software-Foundation/carbon-aware-sdk/issues/40))
+
+Each of these has configuration requirements which are detailed below.
+You can also visit the [quickstart.md](docs/quickstart.md) guide for a step-by-step
+process for running the CLI locally, deploying the Web API locally, polling the
+API via HTTP requests or generating and using client libraries (Python example).
+
+For more detailed architecture and design decisions around the Carbon Aware SDK,
+refer to the [Architecture directory](./architecture/).
+
+## Carbon Aware Library
+
+The Carbon Aware SDK provides a C\# Client Library with handlers that replicates
```

Comment:
We need to remember removing the aggregator mention in this file when we work on the cleanup task. Something to keep in mind.